### ISSUE TITLE
vagrant-vbguest fails on an opensuse image

### ISSUE THREAD

**Issue by andymwood** [dotless-de/vagrant-vbguest#358]
Vagrant version: 2.2.5
vagrant-vbguest version: 0.21.0
VirtualBox version: 6.0.12

To reproduce:
```bash
vagrant init bento/opensuse-leap-15.1
vagrant up
```
which gives the following output:
```text
Bringing machine 'default' up with 'virtualbox' provider...
...
==> default: Machine booted and ready!
[default] GuestAdditions seems to be installed (6.0.12) correctly, but not running.
service: no such service vboxadd
service: no such service vboxadd-service
bash: line 4: setup: command not found
==> default: Checking for guest additions in VM...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

 setup

Stdout from the command:



Stderr from the command:

bash: line 4: setup: command not found
```

I can work around the problem by running the following afterwards:
```bash
vagrant vbguest --do install
vagrant reload
```

**Comment by sidneygijzen** [comment]
I ran into the same error with an `ubuntu/bionic64` box, version 20191125.0.0. Your workaround also solved the problem in my case. Thanks!

Vagrant version: 2.2.6
vagrant-vbguest version: 0.21.0
VirtualBox version: 5.2.34

**Comment by fnordfish** [comment]
fix released in 0.22.0
